### PR TITLE
[MOT] add python implementation for flow_statistics

### PR DESCRIPTION
**Pull request by FlyingQianMM**
At each frame, count the following data from the the beginning to current frame:

* the number of identifiers
* the number of identifiers across the horizontal center line in a top to down manner
* the number of identifiers across the horizontal center line in a down to top manner

At a manual-set interval (unit: seconds), count the number of identifiers from the the beginning to end frame.

### CODE REVIEW

File: deploy/pptracking/python/mot_jde_infer.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def predict_video(detector, camera_id):
         os.makedirs(FLAGS.output_dir)
     out_path = os.path.join(FLAGS.output_dir, video_name)
     if not FLAGS.save_images:
-        fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+        fourcc = cv2.VideoWriter_fourcc(* 'mp4v')
```
Comment (nemonameless) [inline deploy/pptracking/python/mot_jde_infer.py]: *'mp4v'

File: deploy/pptracking/python/mot_jde_infer.py [hunk 2]
```diff
@@ -233,6 +243,24 @@ def predict_video(detector, camera_id):
...
+                out_id_list, prev_center, records, data_type, num_classes)
+            id_set = statistic['id_set']
+            interval_id_set = statistic['interval_id_set']
+            in_id_list = statistic['in_id_list']
+            out_id_list = statistic['out_id_list']
+            prev_center = statistic['prev_center']
+            records = statistic['records']
+
+        elif num_classes > 1 and do_entrance_counting:
+            print('Multi-class flow counting is not implemented now!')
```
Comment (jerrywgz) [inline deploy/pptracking/python/mot_jde_infer.py]: 这里直接报错吧

File: deploy/pptracking/python/utils.py [hunk 3]
```diff
@@ -120,6 +120,16 @@ def argsparser():
         type=int,
         default=50,
         help="max batch_size for reid model inference.")
+    parser.add_argument(
```
Comment (jerrywgz) [inline deploy/pptracking/python/utils.py:123]: 说明下现在只支持单类别

File: deploy/pptracking/python/mot_jde_infer.py (hunk 1 above)
Comment (FlyingQianMM) [reply deploy/pptracking/python/mot_jde_infer.py]: 这里是代码风格检查自动修改的

File: deploy/pptracking/python/mot_jde_infer.py (hunk 2 above)
Comment (FlyingQianMM) [reply deploy/pptracking/python/mot_jde_infer.py]: Done. Thanks!

File: deploy/pptracking/python/utils.py (hunk 3 above)
Comment (FlyingQianMM) [reply deploy/pptracking/python/utils.py:123]: Done. Thanks!

File: deploy/pptracking/python/mot_jde_infer.py (hunk 1 above)
Comment (FlyingQianMM) [reply deploy/pptracking/python/mot_jde_infer.py]: 已经改回去了。感谢！

**Review by nemonameless — APPROVED** [review]

**Review by jerrywgz — APPROVED** [review]

**Pull request merged**